### ISSUE TITLE
Web Content Accessibility Guidelines (WCAG) Issues (text contrast & alt attributes)

### ISSUE THREAD

**Issue by Ldsmiths** [TNRIS/tnris.org#260]
1.) Division of TWDB text

![image](https://user-images.githubusercontent.com/70979203/94479424-65d54880-019a-11eb-8470-af177a46ac8b.png)

This doesn’t pass WCAG standards – the contrast between font and background color isn’t strong enough for the typeface.

![image](https://user-images.githubusercontent.com/70979203/94479466-75ed2800-019a-11eb-9311-f7f9a67203df.png)

Potential fixes:

![image](https://user-images.githubusercontent.com/70979203/94479505-830a1700-019a-11eb-8708-3bef272286cd.png)

_changes to both_

![image](https://user-images.githubusercontent.com/70979203/94479512-856c7100-019a-11eb-9241-46fe3aa78292.png)

_background is same, text change to true white_ 

2.) Carousel Card Text

![image](https://user-images.githubusercontent.com/70979203/94481233-0a588a00-019d-11eb-9b04-0216a1ac92b5.png)

Some of the text is hard to read over lighter images in the carousel cards. This could be fixed by using darker text on light images or by only using images that are dark in areas text will be overlaid. 

3.) Images missing alt attributes

![image](https://user-images.githubusercontent.com/70979203/94479570-9fa64f00-019a-11eb-8b3a-e4cb234a7b5f.png)

SiteImprove has this flagged a few places on the site, especially on “Texas GIS Forum Wrap-up and Photos” pages.

(This helps with accessibility for screen readers, as well as improving SEO)

List of occurrences: 
![image](https://user-images.githubusercontent.com/70979203/94479932-22c7a500-019b-11eb-81ab-e0ffa2f732f4.png)

**Comment by Ldsmiths** [comment]
4) Link text used for multiple different destinations

Description:
The same link text is used for links going to different destinations. Users might not know the difference if they are not somehow explained.

How to fix it:
Make sure links are distinguishable by just their link texts or WAI-ARIA labels ('aria-labelledby' or 'aria-label') to make it clear that they lead to different destinations.
